### DISCUSSION TITLE
Cucumber LSP configuration: Ignore comments and avoid number alignment to the right

### DISCUSSION THREAD

**Discussion by conradogarciaberrotaran** [cucumber/common#2222] [Visual Studio Code]
Hi, first of all, thanks for mantaining this tool. I use it every day.
I'm using cucumber language server in nvim and I've noticed that the LSP moves comments in between examples to the bottom of the scenario:

```cucumber
...
        Examples:
            | foo | bar |
            | 1   | 2   |
            # Some comment
            | 3   | 4   |


```
It will be reformated as:

```cucumber
... 
    Examples:
      | foo | bar |
      |   1 |   2 |
      |   3 |   4 |
       # Some comment
```

Is there a way to disable this? We have several examples and it's useful to have comments for better understanding.
Also, numbers are aligned to the right, would be nice to be able to disable this as well.

Thanks!

**Comment by conradogarciaberrotaran** [comment]
Just found this test in the gherkin-utils library:
https://github.com/cucumber/gherkin-utils/blob/main/java/src/test/java/io/cucumber/gherkin/utils/pretty/PrettyTest.java#L181

It shouldn't move the comments around, so maybe this is a bug?

**Reply by mpkorstanje** [reply]
First you'll want to check if the version of the language server you are using pulls in the right version of Gherkin utils.

Cucumber is mostly developed by people in their spare time. And the IDE support is especially time consuming. So you'll have to do most of the legwork yourself.

And for more context you'll want to read https://mattwynne.net/new-beginning.

**Reply by conradogarciaberrotaran** [reply]
Thanks for replying so quickly.
I'm using language-server 1.6.0 installed through [Mason](https://github.com/williamboman/mason.nvim).

> Cucumber is mostly developed by people in their spare time. And the IDE support is especially time consuming. So you'll have to do most of the legwork yourself.
And for more context you'll want to read https://mattwynne.net/new-beginning.

Understood, so this is a bug that needs fixing. I can give it a try.
Are PRs being merged?

**Reply by mpkorstanje** [reply]
PRs are being merged. Though it may take a while.

**Comment by mpkorstanje** [comment]
Right aligning numbers was implemented as a feature. But ideally the formatter would preserve what ever original alignement was used regardless of content.

But that would be better disuccesed as a feature request in gherkin utils.

**Reply by conradogarciaberrotaran** [reply]
I understand, for me it's not something critical. If it comes up again, we can add a configuration for it.
